src/docfx/validation/JsonSchemaValidatorExtension.cs

```diff
@@ -37,5 +39,31 @@ public bool IsEnable(FilePath filePath, CustomRule customRule)
 
```

Comment:
if you consider the canonicalVersionOnly flag in GetFileEffectiveMonikers, we should modify the `IsEnable` logic?